frontend/portals/ProductCTASBefore/spec.jsx

```diff
@@ -6,7 +6,7 @@ import ShareButton from '../../components/ShareButton';
 const MockedShareButton = () => (<div>ShareButton</div>);
```

Comment:
at the top of this file the ShareButton is imported. This import is not needed. Please remove it.

Reply:
thanks! removed